apps/omg/lib/omg/state/transaction/payment.ex

```diff
@@ -146,7 +153,7 @@ defimpl OMG.State.Transaction.Protocol, for: OMG.State.Transaction.Payment do
 
   @empty_signature <<0::size(520)>>
 
-  # TODO: dry wrt. Application.fetch_env!(:omg, :tx_types_modules)? Use `bimap` perhaps?
+  # TODO: figure out how to nicely reference markers map from config
```

Comment:
Why not define those reference markers here (in the modules), and list the modules in the config (if you actually need the list of all the markers somewhere)?

Reply:
Or even have that list in another module instead of the config.

Reply:
Here you're https://github.com/omisego/elixir-omg/blob/2e10b31e186faa3e2d9d8a921680b8e7c412319e/apps/omg/lib/omg/wire_format_types.ex